resilience4j/tests/test_resilience4j.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from datadog_checks.dev.utils import get_metadata_metrics
+from datadog_checks.resilience4j import Resilience4jCheck
+
+EXPECTED_PROMETHEUS_METRICS = {
+    "bulkhead.available.concurrent.calls",
+    "bulkhead.max.allowed.concurrent.calls",
+    "bulkhead.max.thread.pool.size",
+    "bulkhead.queue.capacity",
+    "bulkhead.queue.depth",
+    "bulkhead.thread.pool.size",
+    "circuitbreaker.buffered.calls",
+    "circuitbreaker.calls.seconds.count",
+    "circuitbreaker.calls.seconds.max",
+    "circuitbreaker.calls.seconds.sum",
+    "circuitbreaker.calls",
+    "circuitbreaker.failure.rate",
+    "circuitbreaker.max.buffered.calls",
+    "circuitbreaker.state",
+    "ratelimiter.available.permissions",
+    "ratelimiter.waiting.threads",
+    "retry.calls",
+    "timelimiter.calls",
+}
+
+
+@pytest.mark.integration
```

Comment:
```suggestion
pytestmark = [
    pytest.mark.integration,
    pytest.mark.usefixtures("dd_environment"),
]
```
Having something like this is encouraged as well as splitting integrations tests in a separate file.
With that, I guess there won't be a need for the `conftest_local.py`, right?